source/includes/data-modeling/persistence-configuration.rb

```diff
@@ -0,0 +1,130 @@
+# start default modeling
+class Restaurant
+    include Mongoid::Document
```

Comment:
I'm not sure it matters for this purpose or not, but on the Ruby team we've standardized on 2-space indentations.

```suggestion
  include Mongoid::Document
```